libpod/network/netconflist.go

```diff
@@ -268,3 +255,82 @@ func IfPassesFilter(netconf *libcni.NetworkConfigList, filters map[string][]stri
 	}
 	return result, nil
 }
+
+// ComputeUntilTimestamp extracts unitil timestamp from filters
+func ComputeUntilTimestamp(filter string, filterValues []string) (time.Time, error) {
```

Comment:
Move this function to `pkg/util`

Reply:
@Luap99 please be certain about this location. In the coming days I will add a unification PR which will remove code duplication. Maybe it is worth to create a new dir that could be shared between libpod and compat endpoints? Please comment on this.

Reply:
I thought about moving it to this file pkg/util/utils.go. I do not want `libpod/network` to be imported by non network related packages like you do in pkg/domain/filters/containers.go.

Removing code duplication between compat and libpod would definitely be great.

Reply:
pkg/util/utils.go? or pkg/util/filter_helpers.go? There will be quite a few functions there coming (depending on my free time of course to do this).

Reply:
If you want to add more a new file would is good.

Reply:
moved to pkg/util/filters.go this file will grow once this one gets merged.